package.json

```diff
@@ -33,28 +33,25 @@
   "dependencies": {
     "broccoli": "0.12.0",
     "broccoli-concat": "0.0.12",
-    "broccoli-defeatureify": "~0.3.0",
+    "broccoli-defeatureify": "~1.0.0",
     "broccoli-derequire": "0.0.1",
-    "broccoli-es3-safe-recast": "0.0.8",
+    "broccoli-es3-safe-recast": "1.0.0",
```

Comment:
@twokul - For now just ones that we "control" (meaning that are managed by folks that we know follow sane practices).